src/components/components/Footer/Footer.scss

```diff
@@ -0,0 +1,64 @@
+.Footer {
+    padding: 48px 16px;
+    background: var(--color-bunting);
+
+    &__container {
+        margin: 0 auto;
+        display: flex;
+        width: 100%;
+        max-width: 1260px;
+        justify-content: space-between;
+        align-items: center;
+
+        &__logo {
+            img {
+                width: 153.6px;
+                height: 36px;
+            }
+        }
+
+        &__frame {
+            display: flex;
+            align-items: center;
+            padding: 10px;
+            gap: 25px;
+            color: var(--color-white);
+
+            a {
+                text-decoration: none;
+                color: inherit;
+                opacity: .8;
+            }
+        }
+    }
+
+    @media screen and (max-width: 720px) {
+        &__container {
+            flex-direction: column;
+            align-items: flex-start;
+            padding: 0 2em;
+
+            &__frame {
+                padding-top: 2em;
+                flex-direction: column;
+                align-items: flex-start;
+
+                .space {
+                    display: none;
+                }
+            }
+        }
+    }
+
+    @media screen and (max-width: 480px) {
+        &__container {
+            align-items: center;
```

Comment:
as per design, all items in footer section are aligned to the left side (flex-start) in mobile view